gems/smithy/spec/spec_helper.rb

```diff
@@ -80,7 +80,7 @@ def create_plan(modules, model, type, options)
         module_name: modules.join('::'),
         gem_version: options.fetch(:gem_version, '0.1.0'),
         destination_root: options.fetch(:destination_root, Dir.mktmpdir),
-        quiet: ENV.fetch('SMITHY_RUBY_DEBUG', 'false') == 'false'
+        quiet: ENV.fetch('SMITHY_RUBY_QUIET', 'true') == 'true'
```

Comment:
Should this be defaulting to true?  I think we'd want to default it to false.

Reply:
In tests don't we want it off by default?

Reply:
Ahhh, right.  nvm.